gradle/libs.versions.toml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2025-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[versions]
+spring-session = "4.0.0-RC1"
+spring-data = "2025.0.5"
+spring-security = "6.5.6"
+spring-framework = "6.2.12"
+jackson = "2.18.4"
+project-reactor = "2025.0.0"
+mongodb-driver = "5.6.1"
+
+# Code verification libraries
+findbugs = "3.0.2"
+nullaway = "0.12.4"
+google-errorprone-core = "2.36.0"
+
+# Test libraries
+assertj = "3.27.3"
+junit = "5.12.1"
+hamcrest = "3.0"
+jakarta-websocket = "2.1.1"
+jakarta-servlet-api = "6.0.0"
+mockito = "5.16.1"
+logback = "1.5.18"
+
+plugin-spotless = "7.0.2"
+plugin-palantir = "2.58.0"
+plugin-ktfmt = "0.54"
+plugin-test-logger = "4.0.0"
+plugin-errorprone = "4.1.0"
+
+[libraries]
```

Comment:
Not sure all these deps are used (unless you're planning to use them later, ex `spring-webflux`)